backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5760250_sys_gh23784_Additional_AD_Messages.sql

```diff
@@ -0,0 +1,37 @@
+-- Run mode: SWING_CLIENT
+
+-- Value: de.metas.handlingunits.MSG_ManufacturingOrderNotCompleted
+-- 2025-07-08T09:08:09.335Z
+INSERT INTO AD_Message (AD_Client_ID,AD_Message_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,MsgText,MsgType,Updated,UpdatedBy,Value) VALUES (0,545566,0,TO_TIMESTAMP('2025-07-08 09:08:09.095000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'D','Y','Fertigungsauftrag ist nicht abgeschlossen ! Produktionsauftrag-ID: {0}','E',TO_TIMESTAMP('2025-07-08 09:08:09.095000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'de.metas.handlingunits.MSG_ManufacturingOrderNotCompleted')
```

Comment:
1. pls avoid spaces before punctuation mark (i.e. `!`)
2. user does not understand IDs. He/She is aware of document nos.
3. try making them more user friendly, e.g. something like "Manufacturing order {0} is not completed."